### PR TITLE
Added Polkadot Hungary Community

### PR DESCRIPTION
**Pull request by send4t**

### CODE REVIEW

File: registry/polkadot.json [hunk 1]
```diff
@@ -365,5 +365,14 @@
...
+        "isOrganization": true
     }
+
```
Comment (stepanLav) [inline registry/polkadot.json:377]: please, remove the empty line
```suggestion
```

File: registry/polkadot.json [hunk 2]
```diff
@@ -365,5 +365,14 @@
         "shortDescription": "Championing balanced growth through strategic marketing and prudent spending.",
         "longDescription": "Seeking to fund reasonable marketing proposals where the value prop can clearly be defined, and aim to hold recepients of funding accountable for their deliverables.",
         "isOrganization": false
+    },
+{
```
Comment (stepanLav) [inline registry/polkadot.json:369]: please, keep format
```suggestion
    {
```

**Comment by stepanLav** [comment]
Hello!
thank you for the submission, could you provide also proof of authority with system (remark)
based on this [tutorial](https://docs.novawallet.io/nova-wallet-wiki/opengov/delegate-registry), thanks!

**Comment by send4t** [comment]
https://polkadot.subscan.io/extrinsic/20195050-2

We submitted with a multisig through governance proxy but it failed for some reason.

**Comment by stepanLav** [comment]
> https://polkadot.subscan.io/extrinsic/20195050-2
> 
> We submitted with a multisig through governance proxy but it failed for some reason.

That account only has proxy for Governance pallet, but it will count, thank you!

**Review by stepanLav — APPROVED** [review]

**Pull request merged**